app/forms/idv/phone_form.rb

```diff
@@ -38,22 +39,29 @@ def submit(params)
       FormResponse.new(success: success, errors: errors, extra: extra_analytics_attributes)
     end
 
-    def user_has_multiple_phones?
-      @user.phone_configurations.many?
-    end
-
     private
 
     attr_writer :phone, :otp_delivery_preference
 
-    def initial_phone_value(input_phone)
-      initial_phone = input_phone
-      initial_phone ||= begin
-        user_phone = MfaContext.new(user).phone_configurations.take&.phone
-        user_phone if valid_phone?(user_phone, phone_confirmed: true)
+    # @return [Array<string,string>] The international_code and phone values to use.
+    def determine_initial_values(international_code: nil, phone: nil, **_)
```

Comment:
I dunno that I feel too strongly, but should we bother with the extra spread here, or would it make sense for the contract of this method to be specifically only the keyword arguments we care about, even if it is the case that we're passing the full set of parameters?

```suggestion
    def determine_initial_values(international_code: nil, phone: nil)
```

Related, I wonder if we should just pass the arguments we care about, e.g. `**previous_params.slice(:international_code, :phone).symbolize_keys` or a verbose-but-slightly-less-clever `international_code: previous_params[:international_code], phone: previous_params[:phone]`.

Reply:
Agreed, updated in 3f8e802